src/ORM/DataObject.php

```diff
@@ -3735,6 +3735,11 @@ public function searchableFields()
                     if ($schema->fieldSpec($this, $name)) {
                         $fields[] = $name;
                     } elseif ($this->relObject($spec)) {
+                        // Field does not always exist as a real db field (e.g. "Member.Title")
```

Comment:
This is a bit confusing

Should this instead be along the lines of

"This is usually a real field i.e. "Member.Title", though it may be some thing else e.g. <insert example>